src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+
+    RightTriangle(String color, double firstLeg, double secondLeg) {
```

Comment:
The constructor of RightTriangle should be public to allow instantiation from outside the package.

Reply:
OK